### PR TITLE
Add support for registering languages

### PR DESCRIPTION
**Pull request by jescalan**
Fairly straightforward feature, exposing the `registerLanguage` feature from lowlight 😀

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -13,13 +13,21 @@ function attacher(options) {
...
+  var languages = settings.languages
   var name = 'hljs'
   var pos
 
   if (aliases) {
     lowlight.registerAlias(aliases)
   }
 
+  if (languages) {
+    // eslint-disable-next-line guard-for-in
```
Comment (jescalan) [inline index.js]: Disabled eslint here bc this rule doesn't make sense in this context, and adding it in creates some more-or-less untestable surface area that lowers the overall coverage to 98%.

Comment (wooorm) [reply index.js]: Yeah I severely dislike that rule, I typically turn it off in `package.json.xo.rules` when it pops up!

File: readme.md [hunk 2]
```diff
@@ -99,6 +99,11 @@ highlighted.
 Register more aliases (`Object<string | Array.<string>>`, default: `{}`).
 Passed to [`lowlight.registerAlias`][register-alias].
 
+###### `options.languages`
+
+Register more languages (`Object<string | function>`, default: `{}`).
+Each key/value pair passed as arguments to [`lowlight.registerLanguage`][register-language].
```
Comment (wooorm) [inline readme.md]:
```suggestion
Each key/value pair passed as arguments to
[`lowlight.registerLanguage`][register-language].
```

This should soothe the linter

**Comment by jescalan** [comment]
Perfect, thank you for the review @wooorm!

**Comment by wooorm** [comment]
Thanks, released!

**Comment by jescalan** [comment]
👏 thank you! it doesn't look like the latest has been pushed to npm though?

**Comment by wooorm** [comment]
Sure? I see 3.1.0 on npm? https://www.npmjs.com/package/rehype-highlight

**Comment by jescalan** [comment]
Ah got it now - weird my local `npm i` wasn't picking it up. Maybe it was a cache thing

**Comment by wooorm** [comment]
`npm update` seems to work for me in those cases, or explicitly updating the package.json to mention 3.1.0

**Pull request merged**